doc/changelog.rst

```diff
@@ -4,9 +4,42 @@ Version history and changelog
 =============================
 
 
-Version 1.0.0-alpha (2018)
+Version 1.0.0 (2020-06-30)
 --------------------------
 
+Code
+~~~~
+
+- Now requires Fortran 2008
+- Rewrote linked list functionality for caching to instead use (reallocating) arrays
+- "Number of components" marker in rsp_tensor output file now written as 'NUM_COMPONENTS' instead of 'NUM COMPONENTS'
+- Significantly decreased usage of array constructors in function/subroutine arguments
+- Now compiles and runs with most compilers but still problems with some Intel/2018 and Intel/2019 setups
+- Fixed various memory leak/out-of-bounds errors that sometimes would happen
+- OpenRSP now looks for available file units before choosing one to use
+- Disabled internal memory limit and memory bookkeeping, may be reinstated later
+- If a response tensor is large, then if it's printed at the debugging print level, it's broken down into smaller chunks
+- Added stops for various currently unsupported residue calculation setups
+- Fixed a bug concerning testing of perturbation frequencies against excitation energy for residue calculations
+- Removed some unused residue-related routines
+- Calculation setup errors encountered in wrapper routines now cause exit, not just warning and then continuing
+- The "excitation" perturbation in a single residue calculation is now given
+  the label EX1 in the rsp_tensor file; however, its current implementation
+  still results in triplication of the calculation result data due to being
+  treated as having three components when in fact it has got only one
+
+
+Project
+~~~~~~~
+
+- Added contribution guide and authorship process guide
+- Updated pull request template to solicit agreement to contribution terms
+- Various changes to documentation
+
+
```

Comment:
Could you link the titles to each section to diffs in the GitHub UI? Between 1.0.0 and HEAD, and between 1.0.0 and 1.0.0-alpha, that is.

Reply:
And here you mean linking the 3 bullet points below "Project"?

Reply:
My hesitation with introducing links: they will either be long links or we need RST syntax but then I need to remember to edit the RST syntax when pasting this into the GitHub release form. Can we instead make them more descriptive to avoid linking?

Reply:
Let's skip this. Interested users can use the GitHub UI to see the diffs.

Reply:
For the record, I meant this with linking:
```
## [Version 2.0.1] - 2020-05-06

[Version 2.0.1]: https://github.com/dftlibs/xcfun/compare/v2.0.0...v2.0.1
```
But I second your concern that this adds more annoying work.

Reply:
It's useful but we can add this after the release is done and less risk of me messing it up.

Reply:
Seconded